packages/kms-musap-rn/README.md

```diff
@@ -6,11 +6,94 @@
   <br>
 </h1>
 
-This Sphereon KMS wrapper module enables the use of MUSAP KMS functionalities within React Native projects. It extends the capabilities of the [`AbstractKeyManagementSystem`](./node_modules/@veramo/key-manager/src/abstract-key-management-system.ts) by integrating the MUSAP library to align with Veramo's key management functions.
-A Sphereon KMS wrapper implementation that provides functionalities of musap kms for react-native projects.
+## Overview
+This module provides a Key Management System (KMS) wrapper that enables the use of MUSAP KMS functionalities within React Native projects. It extends the capabilities of the AbstractKeyManagementSystem by integrating the MUSAP library to align with Veramo's key management functions. This ensures that key generation, management, and signing operations are handled securely and efficiently.
+
+## Available functions
+
+- listKeys
+- createKey
+- deleteKey
+- sign
 
 ### Installation
+To install the module, use the following command:
 
-```shell
+```bash
 yarn add @sphereon/ssi-sdk-ext.kms-musap-rn
 ```
+
+## Usage
+### Creating a Key
+To create a key, you need to specify the key type and provide a keyAlias as part of the metadata. Here is an example of how to create a key:
+
+```typescript
+import {MusapKey, MusapModule} from "@sphereon/musap-react-native";
+import {MusapKeyManagementSystem} from "@sphereon/ssi-sdk-ext.musap-rn-kms";
```

Comment:
Inconsistencies in import names. Below it is kms-musap-rn here it is musap-rn-kms.

Also imports that do not seem relevant like MusapModule

Reply:
you're right. removed that parameter from the constructor. now we only have an optional param for sscdType. and tested it with the test app